frontend/javascripts/libs/find_bounding_box_in_nd.ts

```diff
@@ -0,0 +1,105 @@
+import { V2 } from "libs/mjs";
+import { NdArray } from "ndarray";
+import { Vector2 } from "oxalis/constants";
+
+/* This module provides functions to find a 2D axis-aligned bounding box
+ * around data within a binary mask (NdArray).
+ * findBBoxInMask is a precise algorithm which traverses over all cells
+ * and computes the bounding box from the values that are 1.
+ *
+ * estimateBBoxInMask is an approximative and fast algorithm that takes an initial
+ * bounding box and grows that box until the borders of it don't contain
+ * 1s in the mask. The bounding box is grown by ${maxError} pixels in each
+ * iteration. In comparison to the precise algorithm, it is multiple orders of
+ * magnitude faster (for a 1024x1024x5 mask with a max error of 100, it is 100 to 200 times
+ * faster).
+ */
+
+export type BoundingBox2D = {
+  min: Vector2;
+  max: Vector2; // exclusive
+};
+
+export function findBBoxInMask(mask: NdArray): BoundingBox2D {
```

Comment:
> I dont know how cwise works. But maybe it can be used here?

maybe, but I think it would only be sugar as cwise cannot reduce the runtime complexity. so, I'd rather stick to the simple for loops :)